docs/docs/guides/web3_migration_guide/providers_migration_guide.md

```diff
@@ -165,7 +166,87 @@ const reconnectOptions: ReconnectOptions = {
 };
 ```
 
-##### Error message for reconnect attempts
+#### IpcProvider
+
+The IPC provider is used in node.js dapps when running a local node. And it provide the most secure connection.
+
+In 1.x, options passed in the constructor should be of type [`WebsocketProviderOptions`](https://github.com/web3/web3.js/blob/1.x/packages/web3-core-helpers/types/index.d.ts#L192). The `WebsocketProviderOptions` interface consists of:
+
+```ts
+export interface WebsocketProviderOptions {
+	host?: string;
+	timeout?: number;
+	reconnectDelay?: number;
+	headers?: any;
+	protocol?: string;
+	clientConfig?: object;
+	requestOptions?: any;
+	origin?: string;
+	reconnect?: ReconnectOptions;
+}
```

Comment:
Its for websockect and not for IPC in 1.x. can you double check and remove it.

Reply:
Thanks for noting that. I fix this now.